### PR TITLE
Fix invalid request failed in measurement

### PR DESCRIPTION
**Pull request by Wenxing-hou**
Fix: #72

### CODE REVIEW

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 1]
```diff
@@ -1575,7 +1575,8 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
         spdm_request_size = sizeof(spdm_get_measurements_request_t);
     }
 
-    for (index = 0; index < SPDM_MAX_SLOT_COUNT * 2; index++) {
+    /*the invalid sot_id is from 3 to 0x0E */
```
Comment (steven-bellock) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1578]: We don't know which slot_ids, from 1 to 7, are valid or invalid.

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c [hunk 2]
```diff
@@ -1805,7 +1806,7 @@ common_test_case_t m_spdm_test_group_measurements[] = {
     {SPDM_RESPONDER_TEST_CASE_MEASUREMENTS_INVALID_REQUEST,
      "spdm_test_case_measurements_invalid_request",
      spdm_test_case_measurements_invalid_request,
-     spdm_test_case_measurements_setup_version_any,
+     spdm_test_case_measurements_setup_version_11,
```
Comment (jyao1) [inline library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1809]: should it also applicable to version 1.2 ?

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c (hunk 1 above)
Comment (jyao1) [reply library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c:1578]: The valid slot can be detected via DIGESTS.

Comment (Wenxing-hou) [reply library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c]: Thanks. I have updated the comment.

File: library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c (hunk 2 above)
Comment (Wenxing-hou) [reply library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c]: Yes. The invalid request is also applicable to version 1.2. I have made it back to `spdm_test_case_measurements_setup_version_any`.

**Pull request merged**